system/jlib/jsocket.cpp

```diff
@@ -2125,15 +2125,27 @@ bool CSocket::check_connection()
         rc = sendto(sock, NULL, 0, 0, &u.sa, ul);
```

Comment:
I am doubtful/curious of a send() of 0 bytes actually doing anything ?

Reply:
It doesn't appear to - but that's how the code used to "work".